src/components/exit-this-page/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Exit this page
+description: Give users a way to quickly exit a service, website or application quickly and safely.
+section: Components
+aliases:
+backlog_issue_id: 213
+layout: layout-pane.njk
+status: Experimental
+statusMessage: This component is currently experimental. <a class="govuk-link" href="#known-issues-and-gaps">You'll need to do your own research</a> to decide whether to add this component to your service.
+---
+{% from "_example.njk" import example %}
+
+Give users a way to quickly exit a service, website or application quickly and safely.
+
+For service journeys, this component must be used together with the pattern to help a user [exit a page quickly](/patterns/exit-a-page-quickly/).
+
+{{ example({group: "components", item: "exit-this-page", example: "default", html: true, nunjucks: true, open: false, size: "s"}) }}
+
+
+## When to use this component
+
+Use the component on pages with sensitive information that could:
+- put someone at risk of abuse or retaliation
+- reveal someone’s plans to avoid or escape from harm
+
+For example, when a potential victim is using a service to help them leave a domestic abuser.
+
+You can use this component on either:
+- all the pages in a service
+- parts of the journey with sensitive information
+
+You can also use this component for standalone content pages, such as dashboards and guidance.
+
+## When not to use this component
+
+Do not use this component if the service or content is unlikely to put a user at risk. See the [Exit a page quickly](/patterns/exit-a-page-quickly/) pattern for examples of at-risk and sensitive topics.
+
+Keep in mind that seeing this component might discourage certain users from using your service. If the user does not identify themselves as being at risk, they might see the button on a service and decide it’s not relevant to them.
+
+## How it works
+
+There are 2 ways to use the exit this page component. You can use HTML or, if you are using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
+
+The Exit this page component has several parts, including:
+- a button, that takes the user to another website
+- a secondary link, to give people that use assistive technologies a way to activate the component
+- a loading overlay, to immediately clears any content off the browser until the next website loads
+
+All parts of this component must be used together and alongside the Exit a page quickly pattern.
```

Comment:
```suggestion
You must use all parts of this component with the Exit a page quickly pattern.
```

Reply:
also, do we want to use single quotes around the names of the pattern and component to differentiate them in the sentence?